dev/loadgen/cmd/benchmark.go

```diff
@@ -27,6 +29,8 @@ import (
 	"github.com/gitpod-io/gitpod/ws-manager/api"
 )
 
+const benchmarkAnnotation = "benchmark"
```

Comment:
should annotation be changed instead to standard way of naming them, like: `gitpod.io/benchmark`?

Reply:
ws-manager will prefix it with gitpod.io/annotation. If you do it yourself, an error will occur.